### PR TITLE
Fix KUBE readme

### PR DESCRIPTION
**Pull request by rgrandl**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+This project follows [Google's Open Source Community
+Guidelines](https://opensource.google/conduct/).
+
+## Contribution process
+
+We follow the [fork and branch workflow][workflow].
+
+There will be three Git repositories involved:
+
+1.  *upstream* - the Service Weaver KUBE repository on GitHub.
```
Comment (mwhittaker) [inline CONTRIBUTING.md:31]: In a couple places, you say `Service Weaver KUBE`. What do you think about saying `weaver-kube` if we're referring to the repository, `Service Weaver Kubernetes deployer` if we're referring to the deployer abstractly, and `weaver kube` if we're referring to the actual `weaver kube` binary?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+All submissions, including submissions by project members, require review. We
+use GitHub pull requests for this purpose. Consult [GitHub Help][github_help]
+for more information on using pull requests.
+
+To make sure changes are well coordinated, we ask you to discuss any significant
+change prior to sending a pull request. To do so, either file a
+[new issue][new_issue] or claim an [existing one][issues].
+
+## What to contribute to?
```
Comment (mwhittaker) [inline CONTRIBUTING.md:162]: I think this `What to contribute to` section is a bit stale? Maybe delete it or update it to be weaver-kube specific?

File: README.md [hunk 3]
```diff
@@ -1,237 +1,41 @@
...
 
 Visit [https://serviceweaver.dev][website] to learn about Service Weaver.
 
 ## Overview
 
-![](internal/docs/kube_architecture.png)
+Figure below shows the high level overview of the `KUBE` deployer. The user
+provides the application binary and a configuration file `config.yaml`. The
+deployer builds a container image for the application, and generates Kubernetes
+resources that enables the application to run in a Kubernetes cluster `deployment.yaml`.
```
Comment (mwhittaker) [inline README.md:13]:
```suggestion
The figure below shows a high level overview of the `weaver kube` deployer. The user
provides an application binary and a configuration file `config.yaml`. The
deployer builds a container image for the application and generates Kubernetes
resources that enable the application to run in a Kubernetes cluster.

File: internal/docs/kube_overview.png
Comment (mwhittaker) [inline internal/docs/kube_overview.png:1]: Great diagram Robert! :art:

**Review by mwhittaker — APPROVED** [review]
Awesome! Excited to see people's reaction to the kube deployer :partying_face:

File: CONTRIBUTING.md (hunk 1 above)
Comment (rgrandl) [reply CONTRIBUTING.md]: Good point.

**Review by rgrandl — COMMENTED** [review]
Thanks Michael!

File: CONTRIBUTING.md (hunk 2 above)
Comment (rgrandl) [reply CONTRIBUTING.md]: Done. Copied blindly from /weaver.

**Pull request merged**